### PR TITLE
[POT] Fixed bug with out port for nodes with multiple outputs

### PR DESCRIPTION
**Pull request by l-bat**
### Details:
 - *Fixed a bug with incorrect statistics collection for nodes with multiple outputs (bug was added in #8643)*

### Tickets:
 - *80664*

### CODE REVIEW

**Comment by AlexKoff88** [comment]
@l-bat, where does attention_osr.pb model comes from? Please note that we cannot redistribute the model trained by 3rd parties.

File: tools/pot/openvino/tools/pot/graph/node_utils.py [hunk 1]
```diff
@@ -192,8 +192,11 @@ def get_quantized_input_key(quantized_node):
     Otherwise, key is tuple (fq_input name, output port number)
     """
     quantized_input = get_node_input(quantized_node, 0)
-    quantized_key = create_node_name(quantized_input)
-    return quantized_key
+    key = quantized_input.fullname
+    if len(quantized_input.out_ports()) > 1:
+        port_number = quantized_node.in_port(0).get_source().out
```
Comment (KodiaqQ) [inline tools/pot/openvino/tools/pot/graph/node_utils.py:197]: Why we should process only 0 port?

Comment (l-bat) [reply tools/pot/openvino/tools/pot/graph/node_utils.py:197]: Because in this function we only process FQ

**Review by AlexKoff88 — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@l-bat, can we run nightly tests for this at least?

**Review by AlexKoff88 — APPROVED** [review]

**Comment by l-bat** [comment]
Nightly 
* VNNI: 840
* FP32: 1252

**Pull request merged**